sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -42,7 +44,11 @@ class OrchestrationController {
   public OrchestrationChatResponse completion() {
     final var prompt = new OrchestrationPrompt("Hello world! Why is this phrase so famous?");
 
-    return client.chatCompletion(prompt, config);
+    final var result = client.chatCompletion(prompt, config);
+
+    log.info("Our trusty AI answered with: {}", result.getContent());
+
+    return result;
```

Comment:
(Opinion)

This is a inconsistent change with other methods of this class.
Not sure this adds to **sample** code quality.

Reply:
> how the response can be evaluated.

I understand. Not a fan of the current nor the proposed state.
Hopefully in the future we can have a better sample code project, that's not about JSON serialization solely.

Reply:
Feel free to improve it 😉 

we also have the related there tickets: https://github.tools.sap/AI/ai-sdk-java-backlog/issues/128 and https://github.tools.sap/AI/ai-sdk-java-backlog/issues/126